code/controllers/subsystem/blackbox.dm

```diff
@@ -182,6 +200,15 @@ SUBSYSTEM_DEF(blackbox)
 	var/datum/DBQuery/query_report_death = SSdbcore.NewQuery("INSERT INTO [format_table_name("death")] (pod, x_coord, y_coord, z_coord, mapname, server_ip, server_port, round_id, tod, job, special, name, byondkey, laname, lakey, bruteloss, fireloss, brainloss, oxyloss, toxloss, cloneloss, staminaloss) VALUES ('[sqlpod]', '[x_coord]', '[y_coord]', '[z_coord]', '[map]', INET_ATON(IF('[world.internet_address]' LIKE '', '0', '[world.internet_address]')), '[world.port]', [GLOB.round_id], '[SQLtime()]', '[sqljob]', '[sqlspecial]', '[sqlname]', '[sqlkey]', '[laname]', '[lakey]', [sqlbrute], [sqlfire], [sqlbrain], [sqloxy], [sqltox], [sqlclone], [sqlstamina])")
 	query_report_death.Execute()
 
+/datum/controller/subsystem/blackbox/proc/Seal()
+	if(sealed)
+		return
+	if(IsAdminAdvancedProcCall())
+		var/msg = "[key_name(usr)] sealed the blackbox!"
+		message_admins(msg)
+		log_admin(msg)
+	log_game("Blackbox sealed")

```

Comment:
Feel like this could just be `log_game("Blackbox sealed[IsAdminAdvancedProcCall() ? " by [key_name(usr)]" : ""]."` instead.

Also `log_admin("[key_name_admin(usr)] sealed the blackbox!")`